### PR TITLE
Remove remaining references to research point generation through power generation

### PR DESCRIPTION
**Pull request by esainane**
## About The Pull Request

Radiation collectors and tesla coils being able to create research points was removed in 92209dd23a4cb15f9152fd54ece7b86c52040258. This stops references to power-based research point generation from appearing.

## Why It's Good For The Game

Research by power is long gone

## Changelog

:cl:
fix: Round setup hints and circuit board descriptions that mention production of research points through power generation no longer appear. This functionality was removed a long time ago.
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — APPROVED** [review]
Fuck.

Thank you

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -141,7 +141,6 @@
 /obj/item/circuitboard/machine/tesla_coil
 	name = "Tesla Controller (Machine Board)"
 	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
-	desc = "You can use a screwdriver to switch between Research and Power Generation."
```
Comment (tralezab) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm:144]: Surely you can give it something for a description

Comment (esainane) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:144]: Radiation collector circuits don't have a description either, if I think of something suitably witty I'll add it

Comment (EricZilla) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:144]: "Does not let you shoot lightning from your hands"

**Pull request merged**